.github/workflows/test-software.eessi.io.yml

```diff
@@ -57,16 +57,14 @@ jobs:
               env | grep ^EESSI | sort
 
               # first check the CPU-only builds for this CPU target
-              echo "just run check_missing_installations.sh (should use easystacks/software.eessi.io/${{matrix.EESSI_VERSION}}/eessi-${{matrix.EESSI_VERSION}}-*.yml)"
-              for easystack_file in $(ls easystacks/software.eessi.io/${{matrix.EESSI_VERSION}}/eessi-${{matrix.EESSI_VERSION}}-eb-*.yml); do
+              echo "just run check_missing_installations.sh (should use easystacks/software.eessi.io/${{matrix.EESSI_VERSION}}/eessi-${{matrix.EESSI_VERSION}}-*.yml with latest EasyBuild release)"
+              for easystack_file in $(EESSI_VERSION=${{matrix.EESSI_VERSION}} .github/workflows/scripts/only_latest_easystacks.sh); do
                   if [ ${{matrix.EESSI_SOFTWARE_SUBDIR_OVERRIDE}} = "x86_64/amd/zen4" ]; then
-                      if grep -q 2022b <<<"${easystack_file}"; then
-                          # skip the check of installed software on zen4 for foss/2022b builds
-                          continue
-                      elif grep -q CUDA <<<"${easystack_file}"; then
-                          # skip the check of install CUDA software in the CPU path for zen4
-                          continue
-                      fi
+                      # Make a temporary EasyStack file where we clean out all 2022b stuff and use that
+                      new_easystack=$(mktemp pruned_easystackXXX --suffix=.yml)
+                      # first clean out the options then clean out the .eb name
+                      sed '/2022b\|12\.2\.0/,/\.eb/{/\.eb/!d}' "${easystack_file}" | sed '/2022b\|12\.2\.0/d' > $new_easystack
```

Comment:
It's a fluke that this passes, it doesn't delete all possible entries. However, the fact that it is needed is a hack anyway for the fact that we don't have the `2022b` gaps filled for Zen4

Reply:
Rebuilding for 2022b is going to be pretty rare though...